examples/webengine/hello-sdl/index.html

```diff
@@ -119,86 +130,548 @@
+                    return;
                 }
+
+                this._hmiLevel = hmiLevel;
             }
 
-            async _sleep (timeout = 1000) {
+            async sendRpcRequest(request) {
                 return new Promise((resolve) => {
-                    setTimeout(resolve, timeout);
+                    this._logRpcRequestMessage(request);
+                    this._sdlManager.sendRpcResolve(request)
+                        .then( (response) => {
+                            this._logRpcResponseMessage(response);
+                            resolve();
+                        })
                 });
             }
 
+            _logRpcRequestMessage(message) {
+                this._logRpcMessage(`APP[${this._appId}] -> SDL`, message);
+            }
+
+            _logRpcResponseMessage(message) {
+                this._logRpcMessage(`SDL -> APP[${this._appId}]`, message);
+            }
+
+            _logRegularMessage(message) {
+                this._logMessage(`APP[${this._appId}]:`, message);
+            }
+
+            _logRpcMessage(prefix, message) {
+                const paramify = function (rpcStruct) {
+                    if (rpcStruct instanceof SDL.rpc.RpcStruct) {
+                        const params = rpcStruct.getParameters();
+                        for (const key in params) {
+                            if (Array.isArray(params[key])) { // could be an array of structs
+                                for (let index = 0; index < params[key].length; index++) {
+                                    params[key][index] = paramify(params[key][index]);
+                                }
+                            } else { // not an array. handle normally
+                                params[key] = paramify(params[key]);
+                            }
+                        }
+                        return params;
+                    }
+
+                    if (Array.isArray(rpcStruct)) { // could be an array of structs
+                        for (let index = 0; index < rpcStruct.length; index++) {
+                            rpcStruct[index] = paramify(rpcStruct[index]);
+                        }
+                        return rpcStruct;
+                    }
+
+                    if (rpcStruct instanceof Object) { // could be an object with nested structs
+                        for (const key in rpcStruct) {
+                            rpcStruct[key] = paramify(rpcStruct[key]);
+                        }
+                    }
+
+                    return rpcStruct; // leave it alone
+                }
+
+                const msg_type = SDL.rpc.enums.MessageType.keyForValue(message.getMessageType());
+                this._logMessage(prefix, `Rpc: "${message.getFunctionId()}" Type: "${msg_type}" Params: ${JSON.stringify(paramify(message))}`);
+            }
+
+            _logMessage(prefix, message, isError = false) {
+                const get_time = function() {
```

Comment:
This isn't the only case where it happens, but please stick to the [Naming Convention Guidelines](https://github.com/smartdevicelink/sdl_javascript_suite/blob/develop/GUIDELINES.md#naming-convention): "Objects, functions, and instances should use camelCase."